build/Targets/Imports.targets

```diff
@@ -498,4 +498,13 @@
     <AdditionalFiles Include="@(PublicAPI)" />
   </ItemGroup>
 
+  <!-- CPS doesn't show these items by default, but we want to show them. -->
```

Comment:
EnableDefaultNoneItems=false is a good thing and shouldn't be disabled. Basically the SDK picks up any file in a given directory and adds it as a None item. This causes some weird issues in the IDE like https://github.com/dotnet/sdk/issues/1157. Basically we shouldn't have done that in the SDK and we should set EnableNoneItems=false by default in the SDK if it wasn't a breaking change.

This change here of adding None globs for the files that we want to show explicitly is preferable.

Note though that once we have WPF support, we would Page items so that line may become unnecessary.